### DISCUSSION TITLE
new python version

### DISCUSSION THREAD

**Discussion by KrawczowaKris** [paketo-buildpacks/feedback#326] [Python Team]
Hi team,
could you please provide an update on when support for Python 3.13 is expected in builder-jammy-base buildpack?

**Comment by dmitry-mightydevops** [comment]
Hi Sophie, is this something you can help us with? @sophiewigmore

**Comment by TheSuperiorStanislav** [comment]
https://github.com/paketo-buildpacks/cpython/pull/828 This pr is blocking update of supported python versions

**Reply by dmitry-mightydevops** [reply]
@brayanhenao is it possible to route this PR
https://github.com/paketo-buildpacks/python/pull/970 to someone who can review/approve it?

**Reply by dmikusa** [reply]
2.24.0 is out. Builders are updating now. Should cut releases automatically when done.
